pkgs/applications/networking/instant-messengers/discord/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
     libPath = stdenv.lib.makeLibraryPath [
         libcxx systemd libpulseaudio
         stdenv.cc.cc alsaLib atk cairo cups dbus expat fontconfig freetype
-        gdk_pixbuf glib gnome2.GConf gtk2 libnotify libX11 libXcomposite
+        gdk_pixbuf glib gnome3.gconf gnome3.gtk2 libnotify libX11 libXcomposite
```

Comment:
Using gtk2 from gnome3 does not really help anything. It also probably should not be there.

Reply:
@jtojnar  What path would be correct for gtk2?

Reply:
`gtk2`, just like `gtk3`.